demo/full/scripts/controllers/PlayerKnobsSettings.jsx

```diff
@@ -33,7 +34,11 @@ const PlayerKnobsSettings = ({
         </span>
       </div>
       <div className="player-knobs-content">
-        <PlaybackRateKnob className="black-knob" player={player} />
+        {
+          lowLatencyMode ?
+            null : // In lowLatencyMode, we take back control of the rate
+            <PlaybackRateKnob className="black-knob" player={player} />
+        }
```

Comment:
/!\ This does not save us from influences between contents.
A "regular" content could have a f***-up rate because it is played just after a buffering low-latency content